src/sagemaker/predictor.py

```diff
@@ -623,32 +623,6 @@ def __call__(self, stream, content_type):
 csv_deserializer = _CsvDeserializer()
 
 
-class BytesDeserializer(object):
```

Comment:
label this as "breaking" since now `from sagemaker.predictor import BytesDeserializer` no longer works